### PR TITLE
Prepare for release v1.6.6

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/10034)

### CODE REVIEW

File: install/kubernetes/quick-install.yaml [hunk 1]
```diff
@@ -650,16 +642,3 @@ spec:
       restartPolicy: Always
       serviceAccount: cilium-operator
       serviceAccountName: cilium-operator
-
```
Comment (joestringer) [inline install/kubernetes/quick-install.yaml:653]: Which version of `helm` did you use to generate this? Just wondering if we will keep deleting and re-adding these lines at the end. I'm not sure which version generates these.

**Review by joestringer — CHANGES_REQUESTED** [review]
I see a lot more chart versions when I search, I think we need to update those too?

```
$ git grep -l "1\." install/
install/kubernetes/cilium/Chart.yaml
install/kubernetes/cilium/charts/agent/Chart.yaml
install/kubernetes/cilium/charts/config/Chart.yaml
install/kubernetes/cilium/charts/config/templates/configmap.yaml
install/kubernetes/cilium/charts/managed-etcd/Chart.yaml
install/kubernetes/cilium/charts/nodeinit/Chart.yaml
install/kubernetes/cilium/charts/operator/Chart.yaml
install/kubernetes/cilium/charts/preflight/Chart.yaml
install/kubernetes/cilium/requirements.yaml
install/kubernetes/cilium/values.yaml
install/kubernetes/quick-install.yaml
```

File: install/kubernetes/quick-install.yaml (hunk 1 above)
Comment (aanm) [reply install/kubernetes/quick-install.yaml:653]: ah, good point. I'm using v3.0.2. Maybe we should pin this somewhere? Where?

Comment (joestringer) [reply install/kubernetes/quick-install.yaml:653]: OK, so long as you're on helm3 I don't expect this to cause more churn going forward. I expect that all Cilium developers will switch over shortly anyway.

**Review by joestringer — APPROVED** [review]

**Pull request merged**